addons/stock_fleet/__init__.py

```diff
@@ -1 +1,6 @@
 from . import models
+
+
+def _enable_dispatch_management(env):
+    picking_types = env['stock.picking.type'].search([('code', 'in', ['incoming', 'outgoing'])])
```

Comment:
That doesn't include pick (for 2 steps delivery) and pack (for 3 steps delivery) but I'm not sure it we should handle that since the setting can be manually edited, it's maybe better to keep the code simpler